### PR TITLE
Hrcpp 262/pull csharp ssl

### PR DESCRIPTION
**Pull request by rigazilla**
Ssl integration for C# hotrod client

### CODE REVIEW

File: nativeTestSuite/SSLTest.cs [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+namespace nativeTestSuite
+{
+    class SSLTest
+    {
+        public static int testMain(string[] args)
+        {
+            ConfigurationBuilder conf = new ConfigurationBuilder();
+            conf.AddServer().Host("127.0.0.1").Port(11222).ConnectionTimeout(90000).SocketTimeout(900);
+            SslConfigurationBuilder sslConfB = conf.Ssl();
+            sslConfB.Enable().ServerCAFile("c:\\tmp\\infinispan-ca.pem");
```
Comment (tristantarrant) [inline nativeTestSuite/SSLTest.cs]: Please commit the pem file as part of the test

**Comment by rigazilla** [comment]
added and rebased

File: nativeTestSuite/SSLTest.cs [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+namespace nativeTestSuite
+{
+    class SSLTest
+    {
+        public static int testMain(string[] args)
+        {
+            ConfigurationBuilder conf = new ConfigurationBuilder();
+            conf.AddServer().Host("127.0.0.1").Port(11222).ConnectionTimeout(90000).SocketTimeout(900);
+            SslConfigurationBuilder sslConfB = conf.Ssl();
+            if (!System.IO.File.Exists("c:\\tmp\\infinispan-ca.pem"))
```
Comment (tristantarrant) [inline nativeTestSuite/SSLTest.cs]: Can't you point to the proper file directly ?

**Comment by rigazilla** [comment]
probably I can arrange the build to copy the file in the same place of the executable and then use a relative path.

**Comment by rigazilla** [comment]
ok done

**Pull request merged**